concordium-contracts-common/src/schema_json.rs

```diff
@@ -956,13 +956,234 @@ impl Fields {
     }
 }
 
-/// Displays a pretty-printed JSON-template of the schema.
+/// Displays a pretty-printed JSON-template of the [`SchemaType`].
 impl Display for Type {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         write!(f, "{}", serde_json::to_string_pretty(&self.to_json_template()).unwrap())
     }
 }
 
+/// Displays the json template of the type indented.
+fn display_json_template_indented(
+    mut out: String,
+    type_schema: &Type,
+    type_schema_name: &str,
+    indent: usize,
+) -> String {
+    out = format!("{}{:>3$}{}:\n", out, "", type_schema_name, indent);
+    out = format!(
+        "{}{:>3$}{}\n",
+        out,
+        "",
+        type_schema.to_string().replace('\n', "\n        "),
+        indent + 2
+    );
+    out
+}
+
+/// Displays a pretty-printed template of the [`VersionedModuleSchema`].
+impl Display for VersionedModuleSchema {
```

Comment:
Could you add an example output to the docs, so we can see what the code is supposed to do?